social_protection/custom_filters.py

```diff
@@ -44,6 +46,27 @@ def load_definition(self, tuple_type: type, **kwargs) -> List[namedtuple]:
         list_of_tuple_with_definitions = self.__process_schema_and_build_tuple(benefit_plan, tuple_type)
         return list_of_tuple_with_definitions
 
+    def apply_filter_to_queryset(self, custom_filters: List[namedtuple], query: QuerySet):
+        """
+        Apply custom filters to a queryset.
+
+        :param custom_filters: List of named tuples representing custom filters.
+        :type custom_filters: List[namedtuple]
+
+        :param query: The original queryset with filters.
```

Comment:
Any specific type of QuerySet is required? Like Queryset[BenefitPlan] ?

Reply:
yes, sth like this. Mostly related to Queryset[Beneficiary] in that case.